package.json

```diff
@@ -55,7 +55,7 @@
     "apexcharts": "^3.23.1",
     "autoprefixer": "^9.8.6",
     "axios": "0.21.1",
-    "bnc-notify": "^1.6.2",
+    "bnc-sdk": "^3.4.1",
```

Comment:
The reason I switched to the sdk is because we no longer use the notification system provided by `bnc-notify` (and it also conflicted with our own)